### ISSUE TITLE
How can I fix the port numbers for multiple GPU backends? I don't want to change the port numbers every time I restart the backend

### ISSUE THREAD

**Issue by Ch-Shi** [mcmonkeyprojects/SwarmUI#174]
### Your question

![PixPin_2024-08-17_03-54-14](https://github.com/user-attachments/assets/266099e5-ecfd-4964-a055-4814b3f74519)
I have multiple GPUs and I want to fix the port number and don't want to change it every time.

**Comment by mcmonkey4eva** [comment]
Generally you shouldn't touch the backend ports directly at all? So main question is, why are you even doing that? You're probably doing something wrong if so.

**Comment by Ch-Shi** [comment]
> Generally you shouldn't touch the backend ports directly at all? So main question is, why are you even doing that? You're probably doing something wrong if so.

I found that the port number would change every time I restarted the backend service. Because I was making a workflow using NetDis, I needed to frequently install custom nodes during the process. Each installation required restarting the backend service, which caused the Comfyui port number to change. This invalidated my workflow and required me to re-set the port number in NetDis.
